README.md

```diff
@@ -2587,6 +2587,10 @@ All web server requests include an API version in the url. The current version i
 This endpoint accepts a POST request with a `multipart/form-data` encoded body.  The form fields that may be included are:
 
 * `workflowSource` - *Required* Contains the workflow source file to submit for execution.
+* `workflowType` - *Required* The type of the `workflowSource`, for example "WDL".
```

Comment:
ToL can't we default to Wdl if not specified?

Reply:
We _could_ default the type and type-version, but I believe tech-talk discussions said that the point of bumping to `v2` was the requirement that these be specified. If one doesn't want to specify the values, `v1` will still work (for now).

Reply:
I missed that TT but I recall having the same initial reaction as @danbills when I was caught up. However after chatting w/ @kshakir I remember my thinking changing to the percentage of users actually constructing URLs by hand being quite low, not sure it's worth it.

Reply:
Although if we were to do this we could get away with just keeping `v1` i believe, as then these changes would be backwards compatible